index.html

```diff
@@ -309,6 +335,16 @@
                      readonly="readonly"
                      data-nl-bindto="xpath"
                      data-nl-checkboxbehavior="subscription" />
+              <div class="email-helptip">
+                <i class="far fa-question-circle"></i>
+                <div class="helptip-content">
+                    Need to update your email address or other info? Call us at
+                    <a href="tel:8882787233">(888)278-7233</a>
```

Comment:
@wrandall22 I haven't worked much with telephone links, but should the country code be included?

Reply:
Hmm, maybe so. I'll look into it more.

Reply:
I found this article in a quick search for format requirements in case it helps. https://www.elegantthemes.com/blog/wordpress/call-link-html-phone-number

Reply:
And by "should" I was asking if it would be good to include for potential calling from non-US numbers, not if it was required.

Reply:
Yeah, I think I will add it, but in a separate PR.